tract_querier/query_processor.py

```diff
@@ -558,31 +552,40 @@ def visit_Attribute(self, node):
                 (node.lineno, query_name)
             )
 
-    def visit_Num(self, node):
-        if (
-            node.n in
-            self.tractography_spatial_indexing.crossing_labels_tracts
-        ):
-            tracts = (
-                self.tractography_spatial_indexing.
-                crossing_labels_tracts[node.n]
+    def visit_Constant(self, node):
+        if type(node.value) in [int, float, complex]:
```

Comment:
I think this is better done by `isinstance(node.value, (int, float, complex))`  or `isinstance(node.value, numbers.Number`

Reply:
Not sure: I had first tried with `isinstance(node.value, (int, float, complex))` and the `if` was evaluated to `True` for a `node.value` that was a string, and thus the test was failing. I will revisit as time permits.

Reply:
Changed to `isinstance` in e81a797. See the result of the debugging in the below comment.